internal/host/refresh_status_preprocessor.go

```diff
@@ -59,11 +61,46 @@ func newRefreshPreprocessor(log logrus.FieldLogger, hwValidatorCfg *hardware.Val
 
 const validationDisabledByConfiguration = "Validation disabled by configuration"
 
+func deserializeJSONList(jsonString string) ([]string, error) {
+	var list []string
+	if jsonString != "" {
+		err := json.Unmarshal([]byte(jsonString), &list)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return list, nil
+}
+
+func shouldIgnoreValidation(ignoredValidations []string, validationId string) bool {
+	if !common.MayIgnoreValidation(validationId, common.NonIgnorableHostValidations) {
+		return false
+	}
+	if swag.ContainsStrings(ignoredValidations, "all") {
+		return true
+	}
+	return swag.ContainsStrings(ignoredValidations, validationId)
+}
+
 func (r *refreshPreprocessor) preprocess(ctx context.Context, c *validationContext) (map[string]bool, ValidationsStatus, error) {
 	conditions := make(map[string]bool)
 	validationsOutput := make(ValidationsStatus)
+	var ignoredValidations []string
+	var err error
+	if c.cluster != nil && c.cluster.IgnoredHostValidations != "" {
+		ignoredValidations, err = deserializeJSONList(c.cluster.IgnoredHostValidations)
+		if err != nil {
+			logrus.WithError(err).Warn("Unable to deserialize ignored host validations, no validations will be ignored.")
```

Comment:
Why does the flow continue ?

Reply:
The flow has a responsibility to perform validations, I felt that the correct behaviour is to continue as though the setting had not been applied. At the very least, you'll just get the regular failing validations.

Seems wrong to stop the validation flow due to this?